packages/vite/src/generators/vitest/vitest.spec.ts

```diff
@@ -1,11 +1,12 @@
 import 'nx/src/internal-testing-utils/mock-project-graph';
 
 import { createTreeWithEmptyWorkspace } from '@nx/devkit/testing';
-import { Tree } from '@nx/devkit';
+import { readJson, Tree, updateJson } from '@nx/devkit';
 
-import generator from './vitest-generator';
+import { vitestGeneratorInternal } from './vitest-generator';
```

Comment:
`vitestGeneratorInternal` is the entrypoint of the generator while the default export sets `addPlugin` to `false.
It seems that using `vitestGeneratorInternal` is more realistic.